packages/opencensus-core/src/trace/model/types.ts

```diff
@@ -246,7 +246,7 @@ export interface SpanOptions {
   parentSpanId?: string;
 }
 
-export type TraceState = string;
+export type TraceState = string|undefined;
```

Comment:
What would you think about instead making the places that use `TraceState` explicitly include `|undefined` (maybe even with a find/replace)?

I think making a type include `undefined` or `null` is kind of risks confusing people reading the code (and maybe even bugs) because when they see a type that naturally assume it wouldn't be null/undefined.